src/redux/fallbackExplorer.js

```diff
@@ -58,7 +68,14 @@ const getMainnetAssetsFromCovalent = async (
   const data = await getAssetsFromCovalent(chainId, accountAddress, currency);
   if (data) {
     const updatedAt = new Date(data.updated_at).getTime();
-    const assets = data.items.map(item => {
+    const poolsInWallet = store.getState().uniswapLiquidity.liquidityTokens;
+
+    // remove V2 LP tokens
+    const assets = data.items.filter(
+      item => !find(poolsInWallet, ['address', item.contract_address])
+    );
```

Comment:
the performance people is gonna kill you if you keep using lodash array functions 😅

Reply:
lmao true ill change this is from January